modular-docs-manual/files/sample-assemblies/common/attributes.adoc

```diff
@@ -0,0 +1,18 @@
+// AsciiDoc settings
```

Comment:
@emmurphy1 , I'd remove any of the attributes that aren't actually required by this example repo, just for clarity. Unless all of these are somehow used of course. Seems like several are just carried over. Anyway, whatever you decide.